hacspec/src/lib.rs

```diff
@@ -1668,6 +1747,27 @@ mod tests {
         assert_eq!(c_i.declassify(), c_i_tv.declassify());
     }
 
+    #[test]
+    fn test_parse_message_1_invalid_traces() {
```

Comment:
All these tests should check the type of the error, i.e. whether it is `ParsingError`, to be explicit.

Reply:
Agreed, fixed.